lib/jsdoc/src/visitor.js

```diff
@@ -231,6 +231,43 @@ Visitor.prototype.visitNodeComments = function(node, parser, filename) {
 
     var BLOCK_COMMENT = 'Block';
 
+    // Ensure 'get' method in 'Object.defineProperty' is correctly documented
+    // 'Object.defineProperty' is an ExpressionStatement
+    if (node.type === Syntax.ExpressionStatement &&
+        node.expression.callee &&
+        node.expression.callee.type === Syntax.MemberExpression &&
+        node.expression.callee.object.name === 'Object' &&
+        node.expression.callee.property.name === 'defineProperty' &&
+        node.expression.arguments[2].properties) {
+        // node.expression.arguments[2] is the definition object passed to 'defineProperty'
+        node.expression.arguments[2].properties.forEach(function (propertyNode) {
+            if (propertyNode.key.name !== 'get') {
+                return;
+            }
+
+            var commentNode;
+            var propertyName = node.expression.arguments[1].value;
+            var newComments = '*\n* @name ' + propertyName +
+                '\n* @instance\n';
+
+            // If there are no comments at all no documentation will be generated, so
+            // inject a comment node if necessary
+            if (!propertyNode.leadingComments) {
+                propertyNode.leadingComments = [{
+                    loc: propertyNode.loc,
+                    range: propertyNode.range,
+                    type: 'Block',
+                    raw: '',
+                    value: ''
+                }];
+            }
+
+            commentNode = propertyNode.leadingComments[0];
+            commentNode.value = newComments + commentNode.value;
```

Comment:
Prepending the new comments allows existing comments to override them, but since putting JSDoc tags before the description is invalid we will have to move to this approach. We can simply check for the presence of existing comments and skip adding new ones if necessary.